src/QMCWaveFunctions/tests/test_multi_slater_determinant.cpp

```diff
@@ -96,6 +96,7 @@ void test_LiH_msd(const std::string& spo_xml_string,
   elec_.update();
 
   auto& twf(*twf_ptr);
+  CHECK(twf.findMSD());
```

Comment:
It would be nice to see that given a twf with more than one wfc that we get the MSD as expected with no troubles.  But this is better than nothing.  We have coverage of both sides of the findMSD just not over the loop. This would mostly be to cover future refactoring and possible regression.

Reply:
A second thought. Probably it is better to return a vector of reference wrapper and let the user decide how to handle if there is more than 1. In real simulations, I cannot imaging having two fermionic components in a single TWF.